lib/Dialect/FIRRTL/Transforms/GrandCentral.cpp

```diff
@@ -1174,6 +1174,32 @@ bool GrandCentralPass::traverseField(Attribute field, IntegerAttr id,
         if (!tpe.getBitWidthOrSentinel())
           return true;
 
+        // Generate the path from the LCA to the module that contains the leaf.
+        path += " = ";
+
+        // There are two posisibilites for what this is tapping:
+        //   1. This is a constant that will be synced into the mappings file.
+        //   2. This is something else and we need an XMR.
+        // Handle case (1) here and exit.  Handle case (2) following.
+        auto uloc = builder.getUnknownLoc();
+        auto driver = getDriverFromConnect(leafValue);
+        if (driver) {
+          if (auto constant =
+                  dyn_cast_or_null<ConstantOp>(driver.getDefiningOp())) {
```

Comment:
Good thinking!

Reply:
Err... `getDriverFromConnect` can return `nullptr`, so I don't think this will work. It would likely be better if `getDriverFromConnect` returned the defining op as opposed to the value.